ui/src/models/tag.js

```diff
@@ -0,0 +1,17 @@
+import ApiFetch from '../components/utils/apifetch';
+
+export function searchTag(searchString) {
+    const options = {
+        url: `/api/tags?search=${searchString}`
+    };
+
+    return ApiFetch(options);
+}
+
+export function getPopular() {
+    const options = {
+        url: `/api/tags/popular`
```

Comment:
backtics? Could be normal single quotes right?

Reply:
it could be, doesnt matter, it all boils down to single in the end. :)